app/src/main/java/org/wikipedia/activitytab/ImpactModule.kt

```diff
@@ -300,6 +300,66 @@ fun AllTimeImpactCard(
     }
 }
 
+@Composable
+fun TotalEditsCard(
+    modifier: Modifier = Modifier,
+    uiState: UiState<Int>,
```

Comment:
Since this is a "Card" composable function, can we move the `uiState` logic to `ImpactModule` and handle the network logic there?

```
fun ImpactModule(
    modifier: Modifier = Modifier,
    uiState: UiState<GrowthUserImpact>,
    totalEditsUiState: UiState<Int>,  // for total edits card
    wikiErrorClickEvents: WikiErrorClickEvents? = null,
    totalEditsWikiErrorClickEvents: WikiErrorClickEvents? = null // error event for total edits card
) {
...

```


The idea is to let the `ImpactModule` have the UI state logic and you'll only need to include one `ImpactModule` in `ActivityTabFragment`.